### PR TITLE
Added a new topic to script-topics.rpy

### PR DESCRIPTION
**Pull request by strielok**
This is my first topic so I could have screwed up Monika's expressions a bit.
The dialogue is Monika asking the player about his age, with 3 options to choose from.
Hope it makes it in!

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4782,3 +4782,47 @@ label monika_asks_family:
...
+       m 1m "It's okay though, I still love you..."
+       m 1p "But I really am worried that by the time I get there, you won't be alive anymore..."
+       m 1q "Knowing that I was too late to see you and even talk to you would be devastating."
+       m 1g "So please, [player], try your best to make your life as long as you can."
+       m 1g "If you haven't yet, quit smoking or drinking alcohol and start exercising. I beg you."
+       m 1e "But hey..."
+       m 1e "Maybe by the time I get to you, there will be technology allowing humans to be forever young, or never die."
+       m 2 "You never know!"
+       m 3 "But remember, I'll always love you, regardless of your age, [player]".
+       return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Remove an indentation level from this `return`.

**Comment by strielok** [comment]
Is this what you meant?

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -4782,3 +4782,47 @@ label monika_asks_family:
...
+       m 1m "It's okay though, I still love you..."
+       m 1p "But I really am worried that by the time I get there, you won't be alive anymore..."
+       m 1q "Knowing that I was too late to see you and even talk to you would be devastating."
+       m 1g "So please, [player], try your best to make your life as long as you can."
+       m 1g "If you haven't yet, quit smoking or drinking alcohol and start exercising. I beg you."
+       m 1e "But hey..."
+       m 1e "Maybe by the time I get to you, there will be technology allowing humans to be forever young, or never die."
+       m 2 "You never know!"
+       m 3 "But remember, I'll always love you, regardless of your age, [player]".
+ return
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Please refer to [here](https://github.com/Backdash/MonikaModDev#adding-content) for an example of proper indentation

**Comment by strielok** [comment]
Sorry if im uh, wasting your time, but is this what you meant?

**Comment by MrTwilight** [comment]
I did not like it if you are smaller than she thinks you're a teenager Instead of an elementary school student because if you are about her age then you are a teenager.

**Comment by aldoram5** [comment]
You still have indentation issues your newest pull request, I suggest you re-read https://github.com/Backdash/MonikaModDev/blob/master/CONTRIBUTING.md#making-dialogue
and look how the other topics are indented.

**Comment by strielok** [comment]
is uh, this?

**Comment by ThePotatoGuy** [comment]
Quick example of how your indentation should be like:
```renpy
init 5 python:
    addEvent(.....)

label monika_label_name:
    m "this is a dialogue"
    menu:
        m "this is a question?"
        "response 1":
            m "i saw response 1"
        "response 2":
            m "I saw response 2"
    return
```

**Comment by strielok** [comment]
I'm sorry for wasting your time on this, but, is it good now?
I've did it line by line, so.

**Comment by strielok** [comment]
i presume this is dead

**Comment by monikLover** [comment]
Add in code were player can put in their age and Monika responds accordingly. Closing for now.

**Pull request closed without merging**